src/spec/expectations.cr

```diff
@@ -185,68 +185,90 @@ module Spec
     end
   end
 
+  # This module defines a number of methods to create expectations, which are
+  # automatically included into the top level namespace.
   module Expectations
+    # Creates an `Expectation` that passes if actual equals *value* (`==`).
     def eq(value)
       Spec::EqualExpectation.new value
     end
 
+    # Creates an `Expectation` that passes if actual and *value* are identical (`.same?`).
     def be(value)
       Spec::BeExpectation.new value
     end
 
+    # Creates an `Expectation` that passes if actual is true (`== true`).
     def be_true
       eq true
     end
 
+    # Creates an `Expectation` that passes if actual is false (`== false`).
     def be_false
       eq false
     end
 
+    # Creates an `Expectation` that passes if actual is truthy (neither `nil` nor `false`).
     def be_truthy
       Spec::BeTruthyExpectation.new
     end
 
+    # Creates an `Expectation` that passes if actual is falsy (`nil` or `false`).
     def be_falsey
       Spec::BeFalseyExpectation.new
     end
 
+    # Creates an `Expectation` that passes if actual is nil (`== nil`).
     def be_nil
       eq nil
     end
 
+    # Creates an `Expectation` that passes if actual is within *delta* of *expected*.
     def be_close(expected, delta)
       Spec::CloseExpectation.new(expected, delta)
     end
 
+    # Returns a factory to create a comparison `Expectation` that:
+    #
+    # * passes if actual is lesser than *value*: `be.<(value)`
```

Comment:
Perhaps using the longer `be < value` is clearer here?

Reply:
Yes. For some reason I assumed the `.` was require (dunno why, something failed when I was experimenting with it).